src/Reports/SiteSummary.php

```diff
@@ -32,4 +32,16 @@ public function columns()
     {
         return singleton(Package::class)->summaryFields();
     }
+
+    public function getReportField()
+    {
+        $reportField = parent::getReportField();
+        $reportConfig = $reportField->getConfig()->addComponents(
+            new GridFieldButtonRow('before'),
+            new GridFieldLinkButton('https://addons.silverstripe.org', 'buttons-before-left')
```

Comment:
Can you please use the Injector to create these classes? This will allow third party devs to customise them if they'd like to